test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -48,6 +48,20 @@ Describe "Validate that <pshome>/<culture>/default.help.txt is present" -Tags @(
     }
 }
 
+Describe "Validate that the Help function can Run in strict mode" -Tags @('CI') {
+
+    It "Help doesn't fail when strict mode is on" {
+
+        $help = & {
+            # run in nested scope to keep strict mode from affecting other tests
+            Set-StrictMode -Version Latest
+            Help
+        }
+        # the help function renders the help content as text so just verify that there is content
+        $help | Should -Not -BeNullOrEmpty $true
```

Comment:
`$true` is not needed. See https://github.com/PowerShell/PowerShell/pull/6781#discussion_r185292977

Reply:
Fixed.